ratex/jit/script.py

```diff
@@ -203,16 +204,72 @@ def convert_module_to_raf(module, shape_n_dtype, args):
     ret: Tuple[relay.Function, Dict[str, str], Dict[int, int], Dict[str, raf.array]]
         A tuple of converted function, parameter names, inplace update map, and parameter map
     """
-    model = raf.frontend.from_pytorch(module, {"input0": shape_n_dtype})
+    cloned_module = copy.deepcopy(module)
+
+    bf_fp_16_dtpye_detected = False
+    for (name, para) in cloned_module.named_parameters():
+        if para.dtype == torch.bfloat16:
+            bf_fp_16_dtpye_detected = True
+            bf_fp_16_dtype = "bfloat16"
```

Comment:
This may be undefined so it's not a good practice. I would suggest merging `bf_fp_16_dtype` and `bf_fp_16_dtpye_detected` (btw, you got a typo here) to be like:

```python
bf_fp_16_dtype = None
for ...:
    if ...:
        bf_fp_16_dtype = "bfloat16"
...

if bf_fp_16_dtype is not None:
    cloned_module.to("cpu")
    ...

...
```

Reply:
I have merged those two in the new commits.